source/general/pkteam.rst

```diff
@@ -8,120 +8,138 @@ The ProjectKorra team is comprised of volunteers who work on various aspects of
 
 Meet the Team
 =============
+
+Administrators
+--------------
+Admins are responsible for the day to day operations of ProjectKorra.
+
 +-----------------+-----------------+---------------+
 | Member          | Position        | MC Avatar     |
 +-----------------+-----------------+---------------+
-| Mist            | Admin & Founder | |mist|        |
+| MistPhizzle     | Admin & Founder | |mist|        |
 +-----------------+-----------------+---------------+
 
-Developers
-----------
 
-Developers are the people who maintain and work on the various
-development projects that require programming knowledge
+Dev Team
+--------
+The Dev Team is the backbone of ProjectKorra. Their main responsibilities include maintaining our projects' code and responding to bug-reports. 
+
+If you want to become a developer, click the link below to apply.
+
+`Join the Dev Team! <https://projectkorra.com/join-the-team/>`_
+
+Lead Developers
+^^^^^^^^^^^^^^^
 
 +-----------------+-----------------+---------------+
 | OmniCypher      | Project Lead    | |omnicypher|  |
 +-----------------+-----------------+---------------+
 | Sobki           | Lead Developer  | |sobki|       |
 +-----------------+-----------------+---------------+
+
+Developers
+^^^^^^^^^^
++-----------------+-----------------+---------------+
+| AlexTheCoder    | Developer       | |alexthecoder||
++-----------------+-----------------+---------------+
 | Coolade         | Developer       | |coolade|     |
 +-----------------+-----------------+---------------+
-| Simplicitee     | Developer       | |simplicitee| |
```

Comment:
You removed Simplicitee but never added him back while reordering.

Reply:
that was totally not intentional ;) jk <3